x-pack/plugins/serverless_search/public/application/components/api_key/create_api_key_flyout.tsx

```diff
@@ -138,7 +153,7 @@ export const CreateApiKeyFlyout: React.FC<CreateApiKeyFlyoutProps> = ({
   });
   const createError = parseCreateError(error);
   return (
-    <EuiFlyout onClose={onClose}>
+    <EuiFlyout onClose={onClose} maxWidth="640px">
```

Comment:
question: why not just `size="m"`?

Reply:
This was an addition specifically requested by @julianrosado 

@julianrosado why `maxWidth="640px"` over `size="m"`

Reply:
@sphilipse Because the sizes are all vw percentages, and size m goes way too big and size s goes way too small on desktop. I had not wanted it to get bigger than 640px for line length concerns.

Reply:
I don't know if we need that level of abstraction and complication, since it's less a multiple of a base value and more one very specific max value.
 
If we must then I suggest trolling with `max(calc(calc(${euiTheme.size.xxs} / 2)*640px), 640px);` 👹

Reply:
Eh, just turn it into `calc(${euiTheme.size.xxxl * 10})` or whichever mathy thing you prefer. It's more to be consistent with the rest of the codebase (and so that everything auto-scales nicely if someone decides to tweak the variables).